### PR TITLE
chore: known issues 7.2.0 – March 4

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

Added a couple known issues that won't be addressed in 7.2.0.

# Review requests

Both issues sufficiently and accurately summarized?

### CODE REVIEW

File: app-shell/build/release-notes.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ The Linux version of the Opentrons App now requires Ubuntu 20.04 or newer.
 - The OT-2 now consistently applies tip length calibration. There used to be a height discrepancy between Labware Position Check and protocol runs. If you previously compensated for the inconsistent pipette height with labware offsets, re-run Labware Position Check to avoid pipette crashes.
 - The OT-2 now accurately calculates the position of the Thermocycler. If you previously compensated for the incorrect position with labware offsets, re-run Labware Position Check to avoid pipette crashes.
 
+### Known Issues
+
+- It's possible to start conflicting instrument detachment workflows when controlling one robot from multiple computers. Verify that the robot is idle before starting instrument detachment.
+- Robots may fail to reconnect after renaming them over a USB connection on Windows.
```
Comment (sfoster1) [inline app-shell/build/release-notes.md:32]: This doesn't seem to actually be an issue anymore right @skowalski08

Comment (ecormany) [reply app-shell/build/release-notes.md:32]: I based these on @nusrat813's categorization of RQA-2300 and RQA-2426. Happy to change if either is fixed!

Comment (skowalski08) [reply app-shell/build/release-notes.md:32]: This ticket came up recently:

https://opentrons.atlassian.net/browse/RQA-2300

Rob should be in the office tomorrow, so he can retest.  @sfoster1

Comment (sfoster1) [reply app-shell/build/release-notes.md:32]: Ok, LGTM then

**Review by sfoster1 — APPROVED** [review]
LGTM!

**Comment by ecormany** [comment]
Thanks for getting this into alpha 12.

**Pull request merged**